app/generators/cinema_halls/new_seats.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module CinemaHall
+  module Generators
+    class NewSeats
+
+      def initialize(repository: CinemaHalls::Repository.new)
+        @repository = repository
+      end
+
+      def call
+        seats = CinemaHalls::Generators::GenerateSeats.new(params[:capacity].to_i).call
+        params[:seats] = seats
```

Comment:
It looks like `params` is some kind of a magic variable - where it comes from?

Also, instead of mutating objects, maybe just pass arguments directly?

```
def foo
  params = {foo: "FOO"}
  bar(params)
  params # do you know the output here?
end

def bar(params)
  params[:foo] = "BAR"
  true
end

foo()
```

Reply:
Sure, but you can add new argument in different ways. Like:

```ruby
def call(params)
  another_call(
    foo: params[:foo],
    bar: "BAR"
  )
end
```
or
```ruby
def call(params)
  another_call(
    bar: "BAR",
    **params
  )
end
```
or
```ruby
def call(params)
  another_call(params.merge(bar: "BAR"))
end
```
No need to mutate original argument.

Reply:
@jandudulski for now I decided to delete this method and leave `generate_seats` only.